tests/failingTests.js

```diff
@@ -1,7 +1,7 @@
 window.failingTests = {
   Render: {
     Async: [
-      'NumberAnimationAutorun'
+      'NumberAnimationAutorun', 'Image'
```

Comment:
Um. Did it start failing? It's not failing for me, did something happen? What's the error?